mne/io/nicolet/nicolet.py

```diff
@@ -200,9 +198,8 @@ def _read_segment_file(self, data, idx, offset, fi, start, stop, cals,
             for blk_start in np.arange(0, data_left, blk_size) // nchan:
                 blk_size = min(blk_size, data_left - blk_start * nchan)
                 block = np.fromfile(fid, '<i2', blk_size)
-                block = block.reshape(nchan, -1, order='F')[idx].astype(float)
+                block = block.reshape(nchan, -1, order='F')
                 blk_stop = blk_start + block.shape[1]
-                data[:, blk_start:blk_stop] = block * cal[idx][:, np.newaxis]
-        if mult is not None:
-            data = np.dot(mult, data)
```

Comment:
FYI @jaeilepp this was incorrect before, since `mult` already incorporates the calibration factors `cal` it was being applied twice (and also reconstructed here / ignored from the argument list above?).